src/materials/shaders/pointcloud.vs

```diff
@@ -668,11 +803,13 @@ void doClipping(){
 			//return;
 		//}
 
-		if(time > range.x && time < range.y) {
+		if(!uExtrinsicsMode && (time > range.x && time < range.y)) {
 			//gl_Position = vec4(100.0, 100.0, 100.0, 0.0);
-			vColor.r += .8;
+			vColor.r = 1.0;
```

Comment:
Comment for why 1.0? r make it a named constant, with evocative name?

Reply:
I added a comment in the code. This code here is just setting the point color to red (the shader understands rgb color values ranging from 0 to 1, and 0.8 was a random choice earlier that I decided to just make 1.0)